token-contracts/OMMToken/src/main/java/finance/omm/score/token/OMMTokenImpl.java

```diff
@@ -56,7 +56,7 @@ public void mint(BigInteger _amount, @Optional byte[] _data) {
             throw OMMTokenException.unknown("ZeroValueError: _amount: " + _amount);
         }
 
-        Address to = Context.getCaller();
+        Address to = Address.fromString(Contracts.REWARDS.getKey());
```

Comment:
Contracts.REWARDS.getKey() is not an address. rewrite mint method as follow.

 ```
@External
public void mint(BigInteger _amount, @Optional byte[] _data) {
    Address rewardContract = getAddress(Contracts.REWARDS.getKey());
    onlyOrElseThrow(rewardContract, OMMTokenException.unauthorized("Only reward distribution contract can call mint method"));

    if (_data == null) {
        _data = "minted by reward".getBytes();
    }

    if (_amount.compareTo(BigInteger.ZERO) <= 0) {
        throw OMMTokenException.unknown("ZeroValueError: _amount: " + _amount);
    }

    BigInteger newTotalSupply = this.totalSupply.getOrDefault(BigInteger.ZERO).add(_amount);
    this.totalSupply.set(newTotalSupply);
    BigInteger newBalance = this.balances.getOrDefault(rewardContract, BigInteger.ZERO).add(_amount);
    this.balances.set(rewardContract, newBalance);

    this.Transfer(ZERO_SCORE_ADDRESS, rewardContract, _amount, _data);
}
```

Reply:
Got it